test/emqx_channel_SUITE.erl

```diff
@@ -611,8 +611,12 @@ t_process_alias(_) ->
 t_packing_alias(_) ->
     Packet1 = #mqtt_packet{variable = #mqtt_packet_publish{
                                          topic_name = <<"x">>,
-                                         properties = #{'User-Property' => [{<<"k">>, <<"v">>}]}
-                                        }},
+                                         properties = #{
+                                            'User-Property' => [#{
+                                                key => <<"k">>,
+                                                value => <<"v">>
+                                            }]
+                                        }}},
```

Comment:
... if doing so, these test cases and thus all of the plugins that are potentially using this data format don't have to be changed.